backend/sample_data/database_template.json5

```diff
@@ -25,7 +21,9 @@
                 true,
                 false
             ],
-            "description": "Do your project contain unstructured data?"
+            // TODO: the new question seem to be easy to answer, but how about the accuracy
+            // "description": "Do your project contain unstructured data?",
+            "description": "Do your project have to change the structure of the data regularly?"
```

Comment:
nên đổi tên trường cho sát nghĩa hơn. VD: change_structure_regularly

Reply:
có vẻ thường thì user sẽ không biết là sẽ có sự thay đổi trong data structure.
Vì nếu user biết trước, thì chắc họ cũng thiết kế luôn ngay từ đầu sao cho không phải thay đổi.

Reply:
Bên google họ nói ntn:
Non-relational databases (or NoSQL databases) store complex, unstructured data in a non-tabular form such as documents. Non-relational databases are often used when large quantities of complex and diverse data need to be organized, or where the structure of the data is regularly evolving to meet new business requirements.

Reply:
Thôi để lại là Do your project contain unstructured data? nhé

Reply:
@dduong166 đã fix nhé